roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,9 +250,31 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = [
+        'neutron-dhcp-agent',
+        'neutron-l3-agent',
+        'neutron-metadata-agent',
+        'neutron-linuxbridge-agent'
+    ]
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-d', '--device', help='device to be reset. eg. vxlan')
```

Comment:
s/device to be reset/device type to be reset/ -- presently this looks like it accepts a specific interface name e.g. eth0 or vxlan-1.